CHANGELOG.md

```diff
@@ -29,7 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Fix rendering in 210 tutorial ([#386](https://github.com/etna-team/etna/pull/386))
-- 
+- Remove sorting of `ts.df` ([#389](https://github.com/etna-team/etna/issues/389))
```

Comment:
- This should contain a link to PR, not to issue
- This should give more details
```
Remove sorting of `ts.df` by timestamps in `plot_forecast` and `plot_forecast_decomposition` ([#410](https://github.com/etna-team/etna/pull/410))
```

Reply:
fixed it